rust/tests/integration/consumer.rs

```diff
@@ -993,18 +985,14 @@ fn dump_succeeds() {
                     .consumable_rtp_parameters()
                     .encodings
                     .iter()
-                    .map(|encoding| ConsumableRtpEncoding {
+                    .map(|encoding| RtpEncodingParameters {
                         ssrc: encoding.ssrc,
                         rid: None,
                         codec_payload_type: None,
                         rtx: None,
                         max_bitrate: None,
-                        max_framerate: None,
-                        dtx: None,
+                        dtx: Some(false),
                         scalability_mode: ScalabilityMode::None,
-                        spatial_layers: None,
-                        temporal_layers: None,
-                        ksvc: None
```

Comment:
Hm, why were all these removed? I recall Producer and consumer data structures were distinct for a reason.

Reply:
Nop, they are indeed the same. Verified. Those removed ones are represented by `scalability_mode`.